esmvaltool/diag_scripts/shared/plot/styles/cmip5.style

```diff
@@ -106,4 +107,5 @@ ESACCI-AMSR           |   0,   0,   0        |   1  |   4   |   0  |   0
 NSIDC-NT              |   0,   0,   0        |   2  |   3   |   0  |   0
 NSIDC-BT              |   0,   0,   0        |  12  |   3   |   0  |   0
 HadCRUT4              |   0,   0,   0        |   0  |   3   |   0  |   0
-default               |   0,   0,   0        |   0  |   1   |  16  |   0
+CDS-XCO2              |   0,   0,   0        |   0  |   3   |  16  |   1
+default               |   0,   0,   0        |   0  |   3   |  16  |   0
```

Comment:
Do you need the default option here? This change might also affect other diagnostics.

Reply:
Thanks for spotting! I thought I reverted all the linewidth changes I originally did as I incorporated them in the diagnostic itself now.